README cleanup, tutorial directory

justinGilmer opened the pull request:
Added a directory containing the previous examples.

Created a blank template including all the OpenMM definitions.
Users can remove/add to these as they please

Need to add more about SMARTS however.

justinGilmer committed:
README cleanup, tutorial directory

Added a directory containing the previous examples.

Created a blank template including all the OpenMM definitions.
Users can remove/add to these as they please

Need to add more about SMARTS however.

justinGilmer commented:
Should we include a readme in the tutorial section for SMARTS?

Most of that is being covered in foyer's readmes. `tutorial.md` might not be necessary if thats the case. 

Unsure what would be necessary for the documentation besides that.

ctk3b commented:
Any updates on this? Looks like you've re-arranged it to be template + tutorial as we talked about.

What's left to do from your perspective?

Also one suggestion: adding links to the relevant openmm XML docs for each entry could be convenient.

justinGilmer commented:
Just finishing the markdown document now. 

i went through and have some graphics to show the process to test the examples and what the expected output should be. 

Putting that together now.

justinGilmer committed:
Further improvements to the forcefield_template

Revamped tutorial directory, blank template for user to work through.
Included minimal walk through for tutorial.

Images need work in tutorial, best I could do with my current resources.

Passing forcefield template in top directory.

Relative links where possible in the README's

Image directory for including in tutorial, not all used.

justinGilmer committed:
Clean up .images directory

removed superflous images

justinGilmer committed:
Small grammatical error

ctk3b commented on tutorial/README.md:
````diff
@@ -0,0 +1,267 @@
+emulator of choice within this directory.
+```bash
+py.test -v test_atomtyping.py --tb=line
+```
````
Brief description of options

ctk3b commented on tutorial/README.md:
````diff
@@ -0,0 +1,267 @@
+def="H[C;X4]"
+```
+An important distinction should be pointed out here. In a `SMARTS`
+definition, the first atom in the string is the atom of interest.
````
BOLD AND TRIPLE DOUBLE UNDERLINED
